expat/doc/xmlwf.xml

```diff
@@ -383,19 +383,35 @@ supports both.
 	<command>&dhpackage;</command> prints a single line describing
 	the problem to standard output.  If a file is well formed,
 	<command>&dhpackage;</command> outputs nothing.
-	Note that the result code is <emphasis>not</emphasis> set.
 	</para>
   </refsect1>
+
+  <refsect1>
+  <title>EXIT STATUS</title>
+    <para>The following exit status codes are returned:
+    <variablelist>
+      <varlistentry>
+        <term><option>0</option></term>
+        <listitem><para>The input files are well-formed.</para>
```

Comment:
`xmlwf (-h|-v)` returns 0 and does not fit this case. Should we ignore it?

Reply:
How about I change the text above to say something like: 

> For -h and -v, xmlwf always exits with 0.  For other cases, the exit code is as follows...

then the varlist. OK with you?

Reply:
Seems correct even for cases like `xmlwf -h -Z` (but not `xmlwf -Z -h`). Good enough, I guess. :+1: